grades/models.py

```diff
@@ -102,6 +104,13 @@ def grade_percent(self):
         """Returns the grade field value as a number out of 100 (or None if the value is None)"""
         return self.grade * 100 if self.grade is not None else None
 
+    @property
+    def is_already_combined(self):
+        """
+        Returns true if this grade is for a course run that started after Sept 1st, 2022
+        """
+        return self.course_run.start_date > datetime.datetime(2022, 9, 1, tzinfo=pytz.UTC)
```

Comment:
I think we should convert this date `2022/9/1` into a constant somewhere since we've now used it in multiple places e.g. https://github.com/mitodl/micromasters/blob/master/dashboard/utils.py#L392 and https://github.com/mitodl/micromasters/blob/master/dashboard/utils.py#L570.